pkg/eval/ns_test.go

```diff
@@ -26,6 +26,6 @@ func TestNs(t *testing.T) {
 
 func TestBuiltinFunctionsReadOnly(t *testing.T) {
 	Test(t,
-		That("return~ = { }").Throws(vars.ErrSetReadOnlyVar, "return~"),
+		That("return~ = { }").Throws(ErrorWithType(errs.SetReadOnlyVar{}), "return~"),
```

Comment:
This can test for the value of the error `errs.SetReadOnlyVar{VarName: "return~"}`